applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```
@@ -350,40 +350,40 @@ void BaseSolidElement::CalculateMassMatrix(
         rMassMatrix.resize( mat_size, mat_size, false );
 
     rMassMatrix = ZeroMatrix( mat_size, mat_size );
-    
+
     Matrix DN_DX( number_of_nodes, dimension );
     Matrix J0(dimension,dimension), InvJ0(dimension,dimension);
-    
+
     // Reading integration points and local gradients
     IntegrationMethod integration_method = IntegrationUtilities::GetIntegrationMethodForExactMassMatrixEvaluation(GetGeometry());
     const GeometryType::IntegrationPointsArrayType& integration_points = GetGeometry().IntegrationPoints( integration_method );
     const Matrix& Ncontainer = GetGeometry().ShapeFunctionsValues(integration_method);
 
     double density = 1.0;
-    if ( GetProperties().Has( DENSITY )) 
+    if ( GetProperties().Has( DENSITY ))
```

didn't we say we should throw an error if the density is not there?